provisioner/claim-reconciler/reconiler.go

```diff
@@ -198,3 +196,30 @@ func newBucketConfigMap(ob *v1alpha1.ObjectBucket, obc *v1alpha1.ObjectBucketCla
 		},
 	}
 }
+
+func createUntilDefaultTimeout(obj runtime.Object, c client.Client) error {
+	return wait.PollImmediate(defaultRetryBaseInterval, defaultRetryTimeout, func() (done bool, err error) {
+		err = c.Create(context.Background(), obj)
+		if err != nil && !apierrs.IsAlreadyExists(err) {
+			return false, err
+		}
+		return true, nil
+	})
+}
+
+func extractClaimClass(obc *v1alpha1.ObjectBucketClaim) (string, error) {
+	if obc.Spec.StorageClassName == "" {
+		return "", fmt.Errorf("no class for claim %q", obc.Name)
+	}
+	return obc.Spec.StorageClassName, nil
+}
+
+func translateReclaimPolicy(rp corev1.PersistentVolumeReclaimPolicy) (v1alpha1.ReclaimPolicy, error) {
```

Comment:
Doesn't API validation handle incorrect reclaim policy values (for both Storage Classes and OBs)?